src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -5950,6 +5952,14 @@ void func_8083DDC8(Player* this, PlayState* play) {
 }
 
 void func_8083DF68(Player* this, f32 arg1, s16 arg2) {
+    #ifdef MM_BUNNYHOOD
+
+    if (this->currentMask == PLAYER_MASK_BUNNY) {
+        arg1 *= 1.5f;
```

Comment:
Is it known if this is the exact same behavior as MM?

Reply:
MM makes walking 1.5x faster, but I'm not sure if it slows down rolling. Rando seems to do it, but I don`t see the point behind it. But yeah, if it works differently in MM, we should call it `FASTER_BUNNYHOOD` or something instead.

Reply:
Rolling is just the same speed as normal in MM. This is to prevent cheating probably. Only walking forwards is faster with the bunny hood in MM. Any other action is the same speed.

Reply:
So should we implement MM behavior or should the player always be faster?

Reply:
I’d rather we do whatever MM does for this considering the name of the ifdef. If we can decompile whichever function handles it that would be best so we can recreate it exactly, so this may need a little more research.

Reply:
What about the code @Yanis42 shared earlier? Can we use that until the function is decompiled?